front-end-side/src/Pages/Data.js

```diff
@@ -121,154 +126,239 @@ const EnterData = () => {
+                                        </div>
+                                        {(statusForExpenses && categoryStatus) ? (
+                                            <div id="statistics">
+                                                <div id="cards">
+                                                    <ExpensesCard name="food" categorySum={categoryData.data.foodSum} expenses={newExpenses} />
+                                                    <ExpensesCard name="transportation" categorySum={categoryData.data.carSum} expenses={newExpenses} />
+                                                    <ExpensesCard name="entertainment" categorySum={categoryData.data.entertaintmentSum} expenses={newExpenses} />
+                                                    <ExpensesCard name="house" categorySum={categoryData.data.houseSum} expenses={newExpenses} />
+                                                    <ExpensesCard name="clothes" categorySum={categoryData.data.clothesSum} expenses={newExpenses} />
+                                                    <ExpensesCard name="other" categorySum={categoryData.data.otherSum} expenses={newExpenses} />
+                                                </div>
 
-                                    <div class="double-title">
-                                        <h6 className="date">{datePick.slice(0, 4) + " " + toMonthName(datePick.slice(5, 7))}</h6>
-                                        <h2 className="title">statistics</h2>
-                                    </div>
-                                    {(statusForExpenses && categoryStatus) ? (
-                                        <div id="statistics">
-                                            <div id="cards">
-                                                <ExpensesCard name="food" categorySum={categoryData.data.foodSum} expenses={newExpenses} />
-                                                <ExpensesCard name="transportation" categorySum={categoryData.data.carSum} expenses={newExpenses} />
-                                                <ExpensesCard name="entertainment" categorySum={categoryData.data.entertaintmentSum} expenses={newExpenses} />
-                                                <ExpensesCard name="house" categorySum={categoryData.data.houseSum} expenses={newExpenses} />
-                                                <ExpensesCard name="clothes" categorySum={categoryData.data.clothesSum} expenses={newExpenses} />
-                                                <ExpensesCard name="other" categorySum={categoryData.data.otherSum} expenses={newExpenses} />
+                                                <div id="statistics-graph">
+                                                    <div class="graph-block">
+                                                        <div className="title">{toMonthName(datePick.slice(5, 7))} compared with {toMonthName(datePick.slice(5, 7) - 1)}</div>
```

Comment:
What if user have selected his total expenses? With what graphs are comparing total expenses?

Reply:
Good idea. It will be changed in the future.